### PR TITLE
[sonic-cfggen]: Remove wrong portchannel members

### LINKED ISSUE #9554: [yang-models] portchannel_member validation issue
#### Description

```
 'PORT': {'fortyGigE0/124': {'admin_status': 'up',
                             'description': 'ARISTA04T1:Ethernet1/1',
                             'fec': 'rs',
                             'mtu': '9100',
                             'pfc_asym': 'off',
                             'speed': '100000',
                             'tpid': '0x8100'},
          'fortyGigE0/4': {'admin_status': 'up',
                           'autoneg': 'on',
                           'description': 'Servers0:eth0',
                           'fec': 'none',
                           'mtu': '9100',
                           'pfc_asym': 'off',
                           'speed': '100000',
                           'tpid': '0x8100'}},
 'PORTCHANNEL': {'PortChannel01': {'admin_status': 'up',
                                   'members': ['fortyGigE0/112'],
                                   'min_links': '1',
                                   'mtu': '9100',
                                   'tpid': '0x8100'},
                 'PortChannel02': {'admin_status': 'up',
                                   'members': ['fortyGigE0/116'],
                                   'min_links': '1',
                                   'mtu': '9100',
                                   'tpid': '0x8100'},
                 'PortChannel03': {'admin_status': 'up',
                                   'members': ['fortyGigE0/120'],
                                   'min_links': '1',
                                   'mtu': '9100',
                                   'tpid': '0x8100'},
                 'PortChannel04': {'admin_status': 'up',
                                   'members': ['fortyGigE0/124'],
                                   'min_links': '1',
                                   'mtu': '9100',
                                   'tpid': '0x8100'}},
 'PORTCHANNEL_INTERFACE': {'PortChannel01': {},
                           'PortChannel01|10.0.0.56/31': {},
                           'PortChannel01|FC00::71/126': {},
                           'PortChannel02': {},
                           'PortChannel02|10.0.0.58/31': {},
                           'PortChannel02|FC00::75/126': {},
                           'PortChannel03': {},
                           'PortChannel03|10.0.0.60/31': {},
                           'PortChannel03|FC00::79/126': {},
                           'PortChannel04': {},
                           'PortChannel04|10.0.0.62/31': {},
                           'PortChannel04|FC00::7D/126': {}},
 'PORTCHANNEL_MEMBER': {'PortChannel01|fortyGigE0/112': {'NULL': 'NULL'},
                        'PortChannel02|fortyGigE0/116': {'NULL': 'NULL'},
                        'PortChannel03|fortyGigE0/120': {'NULL': 'NULL'},
                        'PortChannel04|fortyGigE0/124': {'NULL': 'NULL'}},
```
Config db schema generated by minigraph can’t pass yang validation:

* port table does not have fortyGigE0/112, fortyGigE0/116, fortyGigE0/120.

It seems that we need to update yang model or sonic-config-engine.

#### Steps to reproduce the issue:
1. Run command 'sonic-cfggen -m tests/t0-sample-graph.xml --print-data'.
2. Use sonic_yang to validate config db schema.

#### Describe the results you received:
Yang validation failed.

#### Describe the results you expected:
Yang validation should pass.

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

### PR DESCRIPTION
**Pull request by ganglyu**
Signed-off-by: Gang Lv ganglv@microsoft.com

#### Why I did it
Config db schema generated by minigraph can’t pass yang validation, portchannel_member has invalid port.

#### How I did it
Update test minigraph to remove invalid port channel.

#### How to verify it
Build sonic-config-engine.
Run command 'sonic-cfggen -m xxx.xml --print-data', and check port channel member.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

#### Description for the changelog

Fix #9554

### CODE REVIEW

File: src/sonic-config-engine/tests/sample-graph-resource-type.xml [hunk 1]
```diff
@@ -180,11 +180,6 @@
           <AttachTo>fortyGigE0/4</AttachTo>
           <SubInterface/>
         </PortChannel>
-        <PortChannel>
-          <Name>PortChannel1001</Name>
-          <AttachTo>fortyGigE0/1;fortyGigE0/2</AttachTo>
```
Comment (qiluo-msft) [inline src/sonic-config-engine/tests/sample-graph-resource-type.xml:185]: >fortyGigE0/1;fortyGigE0/2 [](http://example.com/codeflow?start=20&length=25)

@neethajohn Did you intend to attach this PortChannel to some existing ports?

File: src/sonic-config-engine/tests/sample-graph-subintf.xml [hunk 2]
```diff
@@ -180,11 +180,6 @@
           <AttachTo>fortyGigE0/4</AttachTo>
           <SubInterface/>
         </PortChannel>
-        <PortChannel>
-          <Name>PortChannel1001</Name>
```
Comment (qiluo-msft) [inline src/sonic-config-engine/tests/sample-graph-subintf.xml:184]: >PortChannel1001 [](http://example.com/codeflow?start=16&length=15)

@neethajohn, the same question.

File: src/sonic-config-engine/tests/simple-sample-graph.xml [hunk 3]
```diff
@@ -180,11 +180,6 @@
           <AttachTo>fortyGigE0/4</AttachTo>
           <SubInterface/>
         </PortChannel>
-        <PortChannel>
-          <Name>PortChannel1001</Name>
-          <AttachTo>fortyGigE0/1;fortyGigE0/2</AttachTo>
```
Comment (qiluo-msft) [inline src/sonic-config-engine/tests/simple-sample-graph.xml:185]: >fortyGigE0/1;fortyGigE0/2 [](http://example.com/codeflow?start=20&length=25)

They are existing ports. Why remove the portchannel?

Comment (ganglyu) [reply src/sonic-config-engine/tests/simple-sample-graph.xml:185]: They are not existing, port name should be: fortyGigE0/0, fortyGigE0/4, fortyGigE0/8...
And this portchannel will not be displayed in config db for invalid member. if I replace member with valid port, config db output will be different, then I need to update test case.
So removing the portchannel seems to be a simple fix.

**Comment by qiluo-msft** [comment]
>       </DeviceLinkBase>

Would you delete the 2 links on non-existing ports?

---
Refers to: src/sonic-config-engine/tests/simple-sample-graph.xml:304 in 46d7b47. [](commit_id = 46d7b470f0b4d2cf2da9fe1cb19143683e22bf72, deletion_comment = False)

**Comment by ganglyu** [comment]
> > ```
> >   </DeviceLinkBase>
> > ```
> 
> Would you delete the 2 links on non-existing ports?
> 
> Refers to: src/sonic-config-engine/tests/simple-sample-graph.xml:304 in [46d7b47](https://github.com/Azure/sonic-buildimage/commit/46d7b470f0b4d2cf2da9fe1cb19143683e22bf72). [](commit_id = [46d7b47](https://github.com/Azure/sonic-buildimage/commit/46d7b470f0b4d2cf2da9fe1cb19143683e22bf72), deletion_comment = False)

will try

**Comment by qiluo-msft** [comment]
>       </DeviceLinkBase>

I did not mean all the DeviceLinkBase. Only the links on non-exsiting fortyGigE0/1 and fortyGigE0/2

---
In reply to: [1000921473](https://github.com/Azure/sonic-buildimage/pull/9614#issuecomment-1000921473) [](http://example.com/codeflow?ancestors=1000921473,1000921473)

---
Refers to: src/sonic-config-engine/tests/simple-sample-graph.xml:304 in 46d7b47. [](commit_id = 46d7b470f0b4d2cf2da9fe1cb19143683e22bf72, deletion_comment = False)

**Review by lguohan — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

File: src/sonic-config-engine/tests/sample-graph-resource-type.xml (hunk 1 above)
Comment (neethajohn) [reply src/sonic-config-engine/tests/sample-graph-resource-type.xml:185]: I had copied simple-sample-graph.xml and made changes for my scenario. This part was untouched.

File: src/sonic-config-engine/tests/sample-graph-subintf.xml (hunk 2 above)
Comment (neethajohn) [reply src/sonic-config-engine/tests/sample-graph-subintf.xml:184]: same comment as above

**Review by neethajohn — APPROVED** [review]

**Pull request merged**